src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -137,17 +93,30 @@ type Client struct {
     metadata map[string][]string
 }
 
-// NewClient creates a new API service client.
-func NewClient(ctx context.Context, opts ...gax.ClientOption) (*Client, error) {
-    s := defaultClientSettings()
+// NewLibraryClient creates a new library service client.
+//
+// This API represents a simple digital library.  It lets you manage Shelf
```

Comment:
@broady 

These comments used to be at the package level for each service. I tried moving them here so they're not lost, so let me know if you want them removed or it they're worth keeping on the constructor.